app/controllers/TeamController.scala

```diff
@@ -50,29 +46,20 @@ class TeamController @Inject()(val messagesApi: MessagesApi) extends Controller{
   def delete(id: String) = SecuredAction.async { implicit request =>
     for {
       team <- TeamDAO.findOneById(id)
-      _ <- (team.owner == request.identity._id) ?~> Messages("team.noOwner")
```

Comment:
why is this line no longer necessary?

Reply:
The accessQuery in the TeamSQLDAO ensures which teams you can delete. However, lines like this might be useful to give usefeul error messages to the user. Their downside is redundancy in the security checks and also additional database queries. I think we should discuss this.